README.md

```diff
@@ -20,6 +20,7 @@ To compile and install the shared library to your system, take the following ste
 mkdir build
 cd build
 cmake ..
+cd ../
```

Comment:
I don't think you want to cd back before calling `make`.

If you want to go back to the root directory, maybe you could try something like this?

```sh
mkdir -p build
pushd build
cmake ..
sudo make -j8 install
popd
```

Reply:
@project-eutopia Oh, it's strange for me, but you are right.
I was wrong.